### PR TITLE
MB-8028-updateMTOShipment-description-match-implementation

### PR DESCRIPTION
**Pull request by NamibiaTorres**
## Description

This ticket fixes a bug on the description of the updateMTOShipment endpoint, that mistakenly tells the user that an agent can not be added but can instead be updated. However, An agent cannot be added nor can they be updated using the `updateMTOShipment` endpoint. Instead, the user should be pointed to the `createMTOAgent` or `updateMTOAgent` endpoints add or update agents, respectively.

A similar update needed to made to the description for addresses to point the user to the `updateMTOShipmentAddress` endpoint, instead of using `updateMTOShipment`. That change in description was made in this PR.

## Reviewer Notes

Please provide feedback on whether the existing validations in the handler and service object are sufficient and do not need to be changed.

## Code Review Verification Steps
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8028) for this change

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

### CODE REVIEW

File: swagger/prime.yaml [hunk 1]
```diff
@@ -217,9 +217,10 @@ paths:
         * `primeEstimatedWeight`
         * `primeActualWeight`
         * `shipmentType`
-        * `agents` - all subfields except `mtoShipmentID`, `createdAt`, `updatedAt`. You cannot add new agents to a shipment.
+        * `agents` - Agents can not be added or updated using this endpoint. For adding use `createMTOAgent` and for updating use `updateMTOAgent`.
```
Comment (shimonacarvalho) [inline swagger/prime.yaml:220]: Hey @NamibiaTorres, I thought we came up with an entirely new description in the documentation meeting. 
Wasn't the goal of that to update this? Maybe I'm mistaken? 

Here's the link @sandy-wright shared after the meeting
https://ustcdp3.slack.com/archives/CP497TGAU/p1621366838012700

**Review by shimonacarvalho — CHANGES_REQUESTED** [review]

File: swagger/prime.yaml [hunk 2]
```diff
@@ -205,19 +205,12 @@ paths:
...
-        * `secondaryDeliveryAddress`
-        * `secondaryPickupAddress`
-        * `primeEstimatedWeight`
-        * `primeActualWeight`
-        * `shipmentType`
-        * `agents` - all subfields except `mtoShipmentID`, `createdAt`, `updatedAt`. You cannot add new agents to a shipment.
+        * Service items: You cannot add or update service items using this endpoint. Please use [createMTOServiceItem](‘#operation/createMTOServiceItem’) and [updateMTOServiceItem](‘#operation/updateMTOServiceItem’) instead.
+        * Agents: You cannot add or update agents using this endpoint. Please use [createMTOAgent](‘#operation/createMTOAgent’) and [updateMTOAgent](‘#operation/updateMTOAgent’) instead.
+        * Addresses: You can add new addresses using this endpoint (and must use this endpoint to do so), but you cannot update existing ones. Please use [updateMTOShipmentAdress](‘#operation/updateMTOShipmentAdress’) instead.
+        These restrictions are due to our [optimistic locking/concurrency control](https://github.com/transcom/mymove/wiki/use-optimistic-locking) mechanism.
```
Comment (shimonacarvalho) [inline swagger/prime.yaml:213]: None of the local links like `operation/updateMTOShipmentAddress` work due to spelling errors and the quotes. Please try this in redoc and check all the links. 
[https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/transcom/mymove/a5044be0c68d5741e06e3a7a5690b2460dca9226/swagger/prime.yaml#operation/updateMTOShipment](redoc for this commit)

Comment (NamibiaTorres) [reply swagger/prime.yaml:213]: > None of the local links like `operation/updateMTOShipmentAddress` work due to spelling errors and the quotes. Please try this in redoc and check all the links. Below is this version in redoc.
> 
> https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/transcom/mymove/a5044be0c68d5741e06e3a7a5690b2460dca9226/swagger/prime.yaml

Thanks @shimonacarvalho! I made the correction and it works now:
https://raw.githubusercontent.com/transcom/mymove/f9e3afe287161783fe7848ae5ad198fc699f2ce0/swagger/prime.yaml

Comment (shimonacarvalho) [reply swagger/prime.yaml:216]: Great works now!

**Review by shimonacarvalho — APPROVED** [review]

**Pull request merged**